---a/net/minecraft/network/play/client/C15PacketClientSettings.java
+++b/net/minecraft/network/play/client/C15PacketClientSettings.java
@@ -1,14 +1,14 @@
 package net.minecraft.network.play.client;
 
 import java.io.IOException;
+
 import net.minecraft.entity.player.EntityPlayer;
 import net.minecraft.network.INetHandler;
 import net.minecraft.network.Packet;
 import net.minecraft.network.PacketBuffer;
 import net.minecraft.network.play.INetHandlerPlayServer;
-import net.minecraftforge.fml.relauncher.Side;
-import net.minecraftforge.fml.relauncher.SideOnly;
 
-public class C15PacketClientSettings implements Packet
-{
+
+public class C15PacketClientSettings implements Packet {
+
     private String lang;
     private int view;
     private EntityPlayer.EnumChatVisibility chatVisibility;
@@ -18,28 +18,10 @@
     private int modelPartFlags;
     private static final String __OBFID = "CL_00001350";
 
-    public C15PacketClientSettings() {}
-
-    @SideOnly(Side.CLIENT)
-    public C15PacketClientSettings(String p_i45946_1_, int p_i45946_2_, EntityPlayer.EnumChatVisibility p_i45946_3_, boolean p_i45946_4_, int p_i45946_5_)
-    {
-        this.lang = p_i45946_1_;
-        this.view = p_i45946_2_;
-        this.chatVisibility = p_i45946_3_;
-        this.enableColors = p_i45946_4_;
-        this.modelPartFlags = p_i45946_5_;
+    public C15PacketClientSettings() {
     }
 
-    public void readPacketData(PacketBuffer buf) throws IOException
-    {
+    public void readPacketData(PacketBuffer buf) throws IOException {
         this.lang = buf.readStringFromBuffer(7);
         this.view = buf.readByte();
         this.chatVisibility = EntityPlayer.EnumChatVisibility.getEnumChatVisibility(buf.readByte());
@@ -39,8 +39,7 @@
         this.modelPartFlags = buf.readUnsignedByte();
     }
 
-    public void writePacketData(PacketBuffer buf) throws IOException
-    {
+    public void writePacketData(PacketBuffer buf) throws IOException {
         buf.writeString(this.lang);
         buf.writeByte(this.view);
         buf.writeByte(this.chatVisibility.getChatVisibility());
@@ -48,34 +48,27 @@
         buf.writeByte(this.modelPartFlags);
     }
 
-    public void processPacket(INetHandlerPlayServer handler)
-    {
+    public void processPacket(INetHandlerPlayServer handler) {
         handler.processClientSettings(this);
     }
 
-    public String getLang()
-    {
+    public String getLang() {
         return this.lang;
     }
 
-    public EntityPlayer.EnumChatVisibility getChatVisibility()
-    {
+    public EntityPlayer.EnumChatVisibility getChatVisibility() {
         return this.chatVisibility;
     }
 
-    public boolean isColorsEnabled()
-    {
+    public boolean isColorsEnabled() {
         return this.enableColors;
     }
 
-    public int getModelPartFlags()
-    {
+    public int getModelPartFlags() {
         return this.modelPartFlags;
     }
 
-    public void processPacket(INetHandler handler)
-    {
-        this.processPacket((INetHandlerPlayServer)handler);
+    public void processPacket(INetHandler handler) {
+        this.processPacket((INetHandlerPlayServer) handler);
     }
-}
+}
